src/containers/NavigationBarContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import { connect } from 'react-redux';
+
+import NavigationBar from '../components/NavigationBar';
+import * as actions from '../redux/game/actions';
+import * as loginActions from '../redux/login/actions';
+import * as globalActions from 'src/redux/global/actions';
+
+import { SiteState } from '../redux/reducer';
+
+const mapStateToProps = (state: SiteState) => ({
+  showRules: state.rules.visible,
+});
+
+const mapDispatchToProps = {
+  newOpenGame: actions.newOpenGame,
```

Comment:
@tomclose Do you think the `newOpenGame` action is needed for the NavBar component?

Reply:
Nope - fixed.